maven-core/src/main/java/org/apache/maven/DefaultMaven.java

```diff
@@ -99,6 +101,9 @@
     @Named( GraphBuilder.HINT )
     private GraphBuilder graphBuilder;
 
+    @Inject
+    private BuildResumptionManager buildResumptionManager;
```

Comment:
In general `Manager` and `Service` are buzzwords, if you don't know what to call it. Not sure it `BuildResumer` better is, but I'd like to avoid Manager.

Reply:
I was also thinking about BuildResumer last time you mentioned that you didn't like the Manager part. I'll make the change, but if others have better names, please suggest them.

Reply:
That being said, "resumer" is not a valid English word, I think.
We could also go for `BuildResumption`?